package.json

```diff
@@ -20,7 +20,7 @@
     "@bubltechnology/customizable-commit-analyzer": "1.0.2-0",
     "@storybook/addon-knobs": "3.1.6",
     "@storybook/addon-notes": "3.1.6",
-    "@storybook/addon-storyshots": "3.1.7",
+    "@storybook/addon-storyshots": "3.1.6",
```

Comment:
no, this is just so they are all on the same version but we still have that same issue with the latest one